packages/instantsearch-ui-components/src/types/Renderer.ts

```diff
@@ -1,11 +1,13 @@
 /* eslint-disable @typescript-eslint/no-namespace */
 /* eslint-disable @typescript-eslint/no-empty-interface */
 
+import type { JSX as ReactJSX } from 'react';
```

Comment:
does this not break for preact then?

Reply:
It shouldn't, but I need to see if TypeScript works with just a simple IS.js project in typescript

I don't know why it worked before, I think it used to load the JSX namespace from `@types/react` but now it doesn't and has a lot of errors saying for example `div` is not in `JSX.IntrinsicElements`

Reply:
That's because the `JSX` namespace used to be wrapped inside a `declare global` but it's not anymore :/